terraform/modules/core/core.tf

```diff
@@ -453,6 +453,10 @@ resource "aws_autoscaling_group" "ecs-autoscaling-group" {
   ]
 
   depends_on = ["aws_vpc.vpc", "aws_launch_configuration.ecs-launch-configuration", "aws_security_group.default", "aws_security_group.ecs"]
+
+  lifecycle {
```

Comment:
This probably needs a comment reflecting the need to comment it out when changing the config.